packages/vx-pattern/test/PatternWaves.test.jsx

```diff
@@ -1,38 +0,0 @@
-import React from 'react';
-import { mount } from 'enzyme';
-
-import { PatternWaves } from '../src';
-
-describe('<PatternWaves />', () => {
-  beforeEach(() => {
-    global.console.error = jest.fn();
```

Comment:
highlighting that `nimbus` mocks out `console.error` to throw, so we don't have to do this ourselves anymore.